extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,22 +85,40 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
-    const children = this.getRealChildren();
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
+        overflowX: 'hidden',
+        webkitOverflowScrolling: 'touch',
```

Comment:
i think moving this to css is super easy , lets do it now rather than refactor it.